docs/track-record-sections.yaml

```diff
@@ -0,0 +1,9 @@
+# Track Record Sections
```

Comment:
Think this file has wrong  filename extension

Reply:
477a0f4